docs/UserGuide.adoc

```diff
@@ -193,11 +207,6 @@ Format: `missappt`
 Settles and removes the missed appointment based on the given index. +
```

Comment:
Settles and removes the missed appointment based on the given index after calling `missappt` cammand.